package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "planout",
-  "version": "2.0.2",
+  "version": "3.0.2",
```

Comment:
I feel pretty strongly about the major version bump here because the opt-in behavior for compatibility mode was to invoke the ```toggleCompatibleHash``` method which no longer exists. Also the default planout.js distribution bundle does not include bignumber in v3, whereas in v2 it did. With v3, consumers will have to choose to grab the compatible bundle instead if they wish for planout core compatibility.